src/storage.js

```diff
@@ -0,0 +1,8 @@
+export const saveStorage = (items) => {
+  localStorage.setItem('todo-list', JSON.stringify(items));
+};
+
+export const getStorage = () => {
+  const tasks = localStorage.getItem('todo-list') ? JSON.parse(localStorage.getItem('todo-list')) : [];
+  return tasks;
+};
```

Comment:
- [ ] The index value should start from 1 but your index value is starting from 2 kindly fix it.
- [ ] The index value is duplicating kindly fix it and when I delete the value the index should update but the index is not updating kindly fix it as well.
### Image
![local storage](https://user-images.githubusercontent.com/81727988/178095899-e9c8c58a-a152-40cb-83f1-adcd2e8cb33a.png)

Reply:
@roshan-bajgain 

Thank you for your feedback

Could you please check again . it is working as expected for me
when I add the tasks it starts from the index 1 
when I delete any task it is being updated.
![Interactive PR](https://user-images.githubusercontent.com/72464136/178096357-09c24865-0aef-40e0-bbfd-934ec09c9179.jpeg)